sdk/identity/identity/review/identity.api.md

```diff
@@ -172,7 +172,7 @@ export class UsernamePasswordCredential implements TokenCredential {
 // @public
 export class VisualStudioCodeCredential implements TokenCredential {
     constructor(options?: VisualStudioCodeCredentialOptions);
-    getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken | null>;
+    getToken(scopes: string | string[]): Promise<AccessToken | null>;
```

Comment:
This is another instance of a deleted options bag. Please restore it! 🙏 Thank you.

Reply:
Everything else looks good 👏

Reply:
options argument is restored in `VisualStudioCodeCredential` class